entity-types/ebpf-redis_server/golden_metrics.yml

```diff
@@ -0,0 +1,24 @@
+throughput:
+  title: Throughput (rpm)
+  unit: REQUESTS_PER_MINUTE
+  query:
+    select: rate(count(redis.latency), 1 minute)
+    from: Span
+responseTimeMS:
+  title: Response time (ms)
+  unit: MS
+  query:
+    select: average(redis.latency)
+    from: Span
```

Comment:
Do we have a guideline on whether we use average, P50, P90, or P99 here?

Reply:
We do compute P50, P90, and P99 on a per protocol basis in the `<protocol>-metrics.pxl` PxL scripts (e.g. [http-metrics.pxl](https://github.com/newrelic/newrelic-ebpf-agent/blob/4bea6e20e50d1e8a047de69fd5099ed9469ef26b/agent-client/scripts/http-metrics.pxl#L64)), though not currently in the `<protocol>-spans.pxl` scripts.

This `golden_metrics.yml` was taken from the existing `ext-pixie_redis` [entity definition](https://github.com/newrelic/entity-definitions/blob/main/entity-types/ext-pixie_redis/golden_metrics.yml).